src/components/GoogleAuth/GoogleAuthStyle.ts

```diff
@@ -27,5 +26,5 @@ export const Typography = styled.div`
 	font-family: Roboto, sans-serif;
 	margin: auto;
 	font-weight: 400;
-	font-size: 16px;
+	font-size: 1.5vw;
```

Comment:
nonono only px and only from constant file

Reply:
fixed